lib/resources.js

```diff
@@ -455,14 +455,17 @@ const buildElasticBody = function (params) {
  * Given a string, returns the string with all unsupported ES control
  * characters escaped. In particular, escapes:
  *
- *  - Specials: '&&', '||', '!', '^', '~', '*', '?', '[', ']', '{', '}', '/'
+ *  - Specials: '&&', '||', '!', '^', '~', '*', '?', '[', ']', '{', '}', '/', '\', '+', '-', '=', '(', ')'
  *  - Colons, except when used in a supported query string query field (e.g. title:Romeo)
  */
 const escapeQuery = function (str) {
   // Escape characters/phrases that should always be escaped:
-  const specials = ['&&', '||', '!', '^', '~', '*', '?', '[', ']', '{', '}']
+  const specials = ['&&', '||', '!', '^', '~', '*', '?', '[', ']', '{', '}', '(', ')', '\\', '+', '-', '=']
```

Comment:
Brackets are escaped here, but they seem to be allowed for making range searches, I'm not sure if that is intended?

Reply:
I think we may have decided to escape them because range queries have to be formatted in a specific, balanced way, and we're not doing anything to detect patrons using valid range queries, so we assume that any use of brackets is not a range query and escape it. That does mean patrons can't run range queries right now, but if we were to support that, I think we'd want to clearly communicate the feature and semantics (e.g. via an Advanced Search page) first. Until then, we treat brackets as a literal part of the query.

Reply:
yeah, I think that's right. In one of our demos earlier this week someone from research was trying to do a search with brackets and getting frustrated that it was returning nothing, so I think escaping helps solve more problems than it creates.